huaweicloud/services/cbh/data_source_huaweicloud_cbh_flavors.go

```diff
@@ -27,6 +27,14 @@ func DataSourceCbhFlavors() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"action": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
```suggestion
				Optional: true,
				Default: "create",
```

Reply:
This is unnecessary, because the API also does not have default behavior.